src/CommandingOfficers/AW1/BH/Sturm.java

```diff
@@ -85,15 +82,7 @@ public void modifyUnitDefenseAgainstUnit(BattleParams params)
   @Override
   public void modifyMoveType(UnitContext uc)
   {
-    for( TerrainType terrain : TerrainType.TerrainTypeList )
-    {
-      final int snowCost = uc.moveType.getMoveCost(Weathers.SNOW, terrain);
-      if( MoveType.IMPASSABLE > snowCost && snowCost > 0 )
-      {
-        uc.moveType.setMoveCost(terrain, 1);
-        uc.moveType.setMoveCost(Weathers.SNOW, terrain, snowCost);
-      }
-    }
+    SturmValueFinders.modifyMoveType(uc);
```

Comment:
Funny syntax. Why is a "finder" doing a "modify" operation. I'm sure there's a good reason, but something isn't named clearly.

Reply:
The "good reason" is that I just dumped the common movement code in the same place all the other Sturm common code is.
I guess I could call it `SturmUtils`.